packages/truffle-contract-schema/typings/index.d.ts

```diff
@@ -0,0 +1,7 @@
+declare module "truffle-contract-schema" {
+  import Contract from "truffle-contract";
+  namespace Schema {
+    export function normalize(dirtyObj: object, opts?: object): Contract.TruffleContract;
```

Comment:
This returns the ContractObject, not a truffle-contract Contract...

i.e. this is more accurate:
```typescript
import { ContractObject } from "truffle-contract-schema/spec";
```